apps/web/src/pages/Auth/SignInPage/SignInPage.tsx

```diff
@@ -49,141 +57,554 @@ const SignInPage = () => {
+                    id: "DL/hpF",
                     description:
-                      "How to get help from the support team - IAP variant",
-                  },
-                  {
-                    a: (chunks: React.ReactNode) =>
-                      buildExternalLink(
-                        "mailto:edsc.pda-iap.esdc@hrsdc-rhdcc.gc.ca",
-                        chunks,
-                      ),
-                  },
-                )}
-              </p>
-            </>
-          )}
-
-          <hr data-h2-margin="base(x2, 0)" />
-          <div
-            data-h2-display="base(flex)"
-            data-h2-flex-direction="base(column) l-tablet(row)"
-            data-h2-align-items="base(center)"
-            data-h2-justify-content="base(space-between)"
-          >
+                      "Seventh answer of the Frequently Asked Questions for logging in",
+                  })}
+                </Accordion.Content>
+              </Accordion.Item>
+              <Accordion.Item value="eight">
+                <Accordion.Header
+                  headingAs="h4"
+                  data-h2-font-size="base(copy)"
+                  data-h2-text-decoration="base(underline)"
+                >
+                  <Accordion.Trigger data-h2-font-weight="base(700)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Can I use SMS or email authentication instead of an app?",
+                      id: "ClrNAj",
+                      description:
+                        "Eighth of the Frequently Asked Questions for logging in",
+                    })}
+                  </Accordion.Trigger>
+                </Accordion.Header>
+                <Accordion.Content>
+                  <Separator
+                    orientation="horizontal"
+                    decorative
+                    data-h2-background-color="base(gray.lighter)"
+                    data-h2-margin="base(x0.5, 0, x1, 0)"
+                  />
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "Currently, our site only supports authentication through an authenticator app.",
+                    id: "CIkGjy",
+                    description:
+                      "Eighth answer of the Frequently Asked Questions for logging in",
+                  })}
+                </Accordion.Content>
+              </Accordion.Item>
+            </Accordion.Root>
             <p>
-              <Link href={!iapMode ? paths.home() : paths.iap()} mode="inline">
-                {intl.formatMessage({
-                  defaultMessage: "Cancel",
-                  id: "OT0QP3",
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    "Read all the FAQ's and still stuck? <helpLink>Contact our team for help</helpLink>",
+                  id: "3M1/4Y",
```

Comment:
Can you add a little margin above this message 😅 `data-h2-margin-top="base(x1)"`

Reply:
I've added this here and to the Sign up page.